purchase_variant_configurator/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    po_confirm_create_variant = fields.Boolean(
+        string="Auto-create product variants",
```

Comment:
This text is not coherent and not changed according https://github.com/OCA/product-variant/pull/362#discussion_r1657599342